ddtrace/tracer/option.go

```diff
@@ -248,6 +251,13 @@ func WithDebugMode(enabled bool) StartOption {
 	}
 }
 
+// WithLambdaMode enables lambda mode on the tracer, for use with amazon lambda.
+func WithLambdaMode(enabled bool) StartOption {
+	return func(c *config) {
+		c.lambda = enabled
```

Comment:
In the other tracers, we default this to true when we detect the env variable 'AWS_LAMBDA_FUNCTION_NAME'.
https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/platform/node/exporter.js